ast/src/main/java/com/graphicsfuzz/common/ast/type/BasicType.java

```diff
@@ -344,26 +344,44 @@ public boolean isBoolean() {
           BVEC4));
   }
 
+  /**
```

Comment:
Hmm, not sure why we have this static method at all.  It looks like numColumns(mat) is the same as mat.getNumColumns().

Can you open an issue for us to delete the static method and change all code that used it to use the instance version?